client/scss/base/_layout.scss

```diff
@@ -5,3 +5,15 @@
     box-sizing: border-box;
   }
 }
+
+.is-mobile-nav-visible {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+
+  @include respond-to('medium') {
```

Comment:
I don't fully understand this respond to - does it mean, "Once you go from small => medium, do this"?